components/Dialogs/examples/DialogsAlertCustomizationViewController.swift

```diff
@@ -83,19 +88,72 @@ class DialogsAlertCustomizationViewController: MDCCollectionViewController {
     switch indexPath.row {
     case 0:
       didTapCenteredTitle()
+    case 1:
+      didTapCenteredTitleWithIcon()
+    case 2:
+      didTapNaturalTitleWithIcon()
+    case 3:
+      didTapRightTitleWithResizedIcon()
+    case 4:
+      didTapNoTitleWithIcon()
     default:
       print("No row is selected")
     }
   }
 
+  func sampleIcon(isStandardSize: Bool = true) -> UIImage? {
+    let bundle = Bundle(for: DialogsAlertCustomizationViewController.self)
+    return UIImage(
+      named: isStandardSize ? "outline_lock_black_24pt" : "baseline_alarm_on_black_48pt",
+      in: bundle, compatibleWith: nil)
+  }
+
   func didTapCenteredTitle() {
-    let alert = createMDCAlertController(title: "Dialog Title")
+    let alert = createMDCAlertController(title: "Center Aligned Title")
+    alert.titleAlignment = .center // todo: theme with themer when available
+    MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
+    self.present(alert, animated: true, completion: nil)
+  }
+
+  func didTapCenteredTitleWithIcon() {
+    let alert = createMDCAlertController(title: "Center Aligned Title")
+    alert.titleIcon = sampleIcon()
     alert.titleAlignment = .center // todo: theme with themer when available
+
+    MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
+    self.present(alert, animated: true, completion: nil)
```

Comment:
_Tip_: Have all the `didTapXYZ` functions return the _alert_ then just call
`self.present(alert, animated: true, completion: nil)` once.